assets/demos/deploy-demoapp.ps1

```diff
@@ -75,7 +75,7 @@ function Deploy-DemoApp {
   if ($Confirmed -eq "y") {
     New-ResourceGroup -Name "benchpress-rg-${EnvironmentSuffix}" -Location $Location -Tags "application=benchpress-demo suffix=${EnvironmentSuffix}"
 
-    New-Deployment -ResourceGroup "benchpress-rg-${EnvironmentSuffix}" -TemplateFile "./assets/demos/main.bicep" -Parameters "suffix=${EnvironmentSuffix}"
+    New-Deployment -ResourceGroup "benchpress-rg-${EnvironmentSuffix}" -TemplateFile "./main.bicep" -Parameters "suffix=${EnvironmentSuffix}"
```

Comment:
Is the assumption that this script needs to be executed from the `assets/demos` directory? If so, the README should be updated to tell the user to `cd` into that directory before running this script.